pkg/get/table.go

```diff
@@ -18,23 +18,35 @@ const (
 // CreateToolTable creates table to show the avaiable CLI tools
 func CreateToolsTable(tools Tools, format TableFormat) {
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"Tool", "Description"})
+
 	table.SetCaption(true,
 		fmt.Sprintf("There are %d tools, use `arkade get NAME` to download one.", len(tools)))
-	if format == MarkdownStyle {
+
+	switch format {
+	case MarkdownStyle:
+		table.SetHeader([]string{"Tool", "Description"})
 		table.SetCaption(true)
 		table.SetBorders(tablewriter.Border{Left: true, Top: false, Right: true, Bottom: false})
 		table.SetCenterSeparator("|")
 		table.SetAutoWrapText(false)
-	} else {
+	default:
+		table.SetHeader([]string{"Tool", "URL", "Description"})
 		table.SetRowLine(true)
 		table.SetColWidth(60)
-		table.SetHeaderColor(tablewriter.Colors{tablewriter.Bold}, tablewriter.Colors{})
-		table.SetColumnColor(tablewriter.Colors{tablewriter.Bold, tablewriter.FgGreenColor}, tablewriter.Colors{})
+		table.SetHeaderColor(tablewriter.Colors{tablewriter.Bold}, tablewriter.Colors{}, tablewriter.Colors{})
+		table.SetColumnColor(tablewriter.Colors{tablewriter.Bold, tablewriter.FgGreenColor}, tablewriter.Colors{}, tablewriter.Colors{})
 	}
 
 	for _, t := range tools {
-		table.Append([]string{t.Name, t.Description})
+		url := fmt.Sprintf("https://github.com/%s/%s", t.Owner, t.Repo)
+
+		switch format {
+		case MarkdownStyle:
+			name := fmt.Sprintf("[%s](%s)", t.Name, url)
```

Comment:
Wait, it looks like you may have done this already.

I think I was confused by the example in the testing section being for the plain format table.

Do we know that every link in arkade has an equivalent URL on GitHub?

Reply:
Yes, I think I've clicked all of them...

Reply:
I think I misunderstood you. But yes, i've done the markdown links for markdown format, separate column for table view and no additional changes just to the list of tools.